client/reader/stream/utils.js

```diff
@@ -38,6 +38,19 @@ export function sameDay( postKey1, postKey2 ) {
 	return moment( postKey1.date ).isSame( postKey2.date, 'day' );
 }
 
+export function sameXPost( postKey1, postKey2 ) {
```

Comment:
`someXPost` -> `isSameXPost`?

Reply:
I prefer that too! It's preceded by two functions called `sameSite` and `sameDate` so I've just gone with `sameXPost` for consistency's sake.